glue_jupyter/bqplot/tests/test_bqplot.py

```diff
@@ -5,6 +5,7 @@
 from nbconvert.preprocessors import ExecutePreprocessor
 from glue.core import Data
 from glue.core.roi import EllipticalROI
+from ..common.tools import TrueCircularROI
```

Comment:
Do we still need relative import in Python 3?

Reply:
Dunno, is it discouraged now?